On macos 10.15:

```
  In file included from crypto/rand/rand_unix.c:38:
  /nix/store/zhfzmfl57q33zc3hw8k3a4gl5hbymzwp-Libsystem-1238.60.2/include/CommonCrypto/CommonRandom.h:35:9: error: unknown type name 'CCCryptorStatus'
  typedef CCCryptorStatus CCRNGStatus;
          ^
  crypto/rand/rand_unix.c:385:47: error: use of undeclared identifier 'kCCSuccess'
      if (CCRandomGenerateBytes(buf, buflen) == kCCSuccess)
                                                ^
  2 errors generated.
  make[1]: *** [Makefile:4496: crypto/rand/rand_unix.o] Error 1
  make: *** [Makefile:177: build_libs] Error 2
  thread 'main' panicked at '


  Error building OpenSSL:
      Command: "make" "build_libs"
      Exit status: exit status: 2


      ', /private/tmp/nix-build-materialize-0.9.4.drv-0/materialize-0.9.4-vendor.tar.gz/openssl-src/src/lib.rs:479:13
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: build failed
```

--- a/pkgs/servers/sql/materialize/default.nix
+++ b/pkgs/servers/sql/materialize/default.nix
@@ -40,21 +40,24 @@ let
 in
 rustPlatform.buildRustPackage rec {
   pname = "materialize";
-  version = "0.8.3";
-  rev = "3e47d280945f0ddbdc263fc643f067649f66eda1";
+  version = "0.9.4";
+  rev = "29d003cae5e9d46f8b11b2102ff0b9abf6608c2f";
 
   src = fetchFromGitHub {
     owner = "MaterializeInc";
     repo = pname;
     rev = "v${version}";
-    sha256 = "15635hcw04nms133k9l5582xbk7frc86nka91yaafl63q6pwbdvg";
+    sha256 = "021n05csyvza9ifq09qaxypgmlbp3a7xn6r1m4jn8d4rnz38wag6";
   };
 
-  cargoSha256 = "1fxqjhjs4bqq6nzax7xrhwjpv57dk90iw9fl8ilp41wzg7yvyhcm";
+  cargoSha256 = "12fysxzmqnx7y7yg6fjcv1952s77d46pwi32vnsv62icgqfpw0j4";
 
   nativeBuildInputs = [ cmake perl pkg-config ]
     # Provides the mig command used by the krb5-src build script
     ++ lib.optional stdenv.isDarwin bootstrap_cmds;
+
+  # Needed to get openssl-sys to use pkg-config.
+  OPENSSL_NO_VENDOR = 1;
 
   buildInputs = [ openssl ]
     ++ lib.optionals stdenv.isDarwin [ libiconv DiskArbitration Foundation ];

